src/app/modules/gb-data-selection-module/gb-data-selection.component.html

```diff
@@ -65,18 +65,27 @@
       <gb-projection></gb-projection>
     </md2-accordion-tab>
 
-    <!--<md2-accordion-tab>-->
-    <!--<md2-accordion-header>-->
-    <!--<span class="gb-data-selection-accordion-header">Step 3: Visualize the selected data (optional)</span>-->
-    <!--</md2-accordion-header>-->
-    <!--Visualization-->
-    <!--</md2-accordion-tab>-->
+    <md2-accordion-tab>
+      <md2-accordion-header>
+        <div class="gb-data-selection-accordion-header">
+          <div class="gb-data-selection-accordion-sub-header-1">
+            Step 3: View and arrange the selected data
+          </div>
+          <div class="form-inline gb-data-selection-accordion-sub-header-2">
+            <div class="gb-data-selection-accordion-sub-header-3">
+              <span class="gb-data-selection-emphasis-text">{{subjectCount_2}} subjects</span>,
+              <span class="gb-data-selection-emphasis-text">{{observationCount_2}} observations</span>
```

Comment:
why these counts here?

Reply:
when the user create or filter the table, there is a further selection of data, which results new counts

Reply:
ah ok, I thought we will have separate counts per each step so a count only includes update from the current step.